### PR TITLE
[BM CPT] Jetlag / add dynamic inventory

### PR DESCRIPTION
**Pull request by josecastillolema**

### CODE REVIEW

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-sno

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-sno

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-sno

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-6nodes

File: ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh [hunk 1]
```diff
@@ -78,4 +81,4 @@ sshpass -p "$(cat /secret/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHost
    cd jetlag
    git branch
    source bootstrap.sh
-   ansible-playbook -i ansible/inventory/telco_${TYPE}.inv ansible/${TYPE}-deploy.yml -v | tee /tmp/ansible-${TYPE}-deploy-$(date +%s)"
\ No newline at end of file
+   ansible-playbook -i ansible/inventory/cloud31.local ansible/${TYPE}-deploy.yml -v | tee /tmp/ansible-${TYPE}-deploy-$(date +%s)"
```
Comment (jtaleric) [inline ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh:84]: lets make this a var - `CLOUD_ID` so we can introduce other clouds in the future.

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-sno

File: ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh [hunk 2]
```diff
@@ -78,4 +81,4 @@ sshpass -p "$(cat /secret/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHost
    cd jetlag
    git branch
    source bootstrap.sh
-   ansible-playbook -i ansible/inventory/telco_${TYPE}.inv ansible/${TYPE}-deploy.yml -v | tee /tmp/ansible-${TYPE}-deploy-$(date +%s)"
\ No newline at end of file
+   ansible-playbook -i ansible/inventory/$LAB_CLOUD.local ansible/${TYPE}-deploy.yml -v | tee /tmp/ansible-${TYPE}-deploy-$(date +%s)"
```
Comment (jtaleric) [inline ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh:84]: Should this be `${LAB_CLOUD}` not sure if bash will throw up here

File: ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh (hunk 1 above)
Comment (josecastillolema) [reply ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh]: Done, thanks for the catch

**Pull request merged**